cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -63,17 +63,17 @@ spec:
   selector:
     matchLabels:
       k8s-app: elasticsearch-logging
-      version: v6.3.0
+      version: v6.6.1
   template:
     metadata:
       labels:
         k8s-app: elasticsearch-logging
-        version: v6.3.0
+        version: v6.6.1
         kubernetes.io/cluster-service: "true"
     spec:
       serviceAccountName: elasticsearch-logging
       containers:
-      - image: k8s.gcr.io/elasticsearch:v6.3.0
+      - image: k8s.gcr.io/elasticsearch:v6.6.1
```

Comment:
this is the old location.  we're not planning on updating this repository any more.  the image should be at:  `gcr.io/fluentd-elasticsearch/elasticsearch`

Reply:
nuts, the other PR only changed the default build location, not what the image repository.  

Do you want to make those changes here or I can push another PR.  Also need to move the fluentd image.

Reply:
changed image to gcr.io/fluentd-elasticsearch/elasticsearch